### PR TITLE
Increase max.partition.fetch.bytes on events-topic consumer

### PR DESCRIPTION
**Pull request by kruai**
# Overview

This PR is being created to address [ESSNTL-3090](https://issues.redhat.com/browse/ESSNTL-3090).
Increases `max.partition.fetch.bytes` on the events-topic consumer. This will allow the host-synchronizer job to consume messages on the events topic that are larger than 1 MB.

## PR Checklist

- [x] Keep PR title short, ideally under 72 characters
- [ ] Descriptive comments provided in complex code blocks
- [ ] Tests: validate optimal/expected output
- [ ] Tests: validate exceptions and failure scenarios
- [ ] Tests: edge cases
- [ ] Recovers or fails gracefully during potential resource outages (e.g. DB, Kafka)
- [ ] Uses [type hinting](https://docs.python.org/3/library/typing.html), if convenient
- [ ] Documentation, if this PR changes the way other services interact with host inventory
- [ ] Links to related PRs

## Secure Coding Practices Documentation Reference

You can find documentation on this checklist [here](https://github.com/RedHatInsights/secure-coding-checklist).

## Secure Coding Checklist

- [ ] Input Validation
- [ ] Output Encoding
- [ ] Authentication and Password Management
- [ ] Session Management
- [ ] Access Control
- [ ] Cryptographic Practices
- [ ] Error Handling and Logging
- [ ] Data Protection
- [ ] Communication Security
- [ ] System Configuration
- [ ] Database Security
- [ ] File Management
- [ ] Memory Management
- [ ] General Coding Practices

### CODE REVIEW

File: app/config.py [hunk 1]
```diff
@@ -162,6 +162,7 @@ def __init__(self, runtime_environment):
             "enable_auto_commit": False,
             "max_poll_records": int(os.environ.get("KAFKA_CONSUMER_MAX_POLL_RECORDS", "10000")),
             "max_poll_interval_ms": int(os.environ.get("KAFKA_CONSUMER_MAX_POLL_INTERVAL_MS", "300000")),
+            "max_partition_fetch_bytes": "3145728",
```
Comment (thearifismail) [inline app/config.py:165]:
```suggestion
            "max_partition_fetch_bytes": int(os.environ.get("KAFKA_CONSUMER_MAX_FETCH_BYTES", "3145728")),
```

Comment (thearifismail) [reply app/config.py:165]: the max value should be set in `deployment.yaml`

**Review by thearifismail — APPROVED** [review]

**Review by FabriciaDinizRH — APPROVED** [review]

**Pull request merged**